app/static/currency.js

```diff
@@ -0,0 +1,54 @@
+function getCurrency(baseCurrency, dateToday) {
+    const showCurrencyElement = document.getElementById("showCurrency")
+    const currencyViewElement = document.getElementById("currencyView");
+    const currencyButtonElement = document.getElementById("currencyButton");
+    currencyViewElement.innerHTML = "";
+
+    function getResponse(yourUrl) {
+        var httpRequest = new XMLHttpRequest();
+        httpRequest.open("GET", yourUrl, false);
+        httpRequest.send(null);
+        return httpRequest.responseText;
+    }
+
+    const data = JSON.parse(getResponse('https://api.exchangeratesapi.io/' + dateToday + '?base=' + baseCurrency));
+    if (!('rates' in data)) {
+        return false;
+    }
+
+    if (showCurrencyElement.style.display == 'none') {
```

Comment:
Prefer using `data-` attributes instead of checking what is the element's style.

Mixing logic and styling is usually not a good idea :)